packages/expo-insights/ios/InsightsModule.swift

```diff
@@ -38,70 +40,70 @@ public final class InsightsModule: Module {
     let data = getLaunchEventData(projectId: projectId)
     try await dispatchEvent(projectId: projectId, eventName: "APP_LAUNCH", data: data)
   }
-}
 
-/**
- Sends an event with the given name and data.
- */
-private func dispatchEvent(projectId: String, eventName: String, data: [String: String?]) async throws {
-  let endpointUrl = "https://i.expo.dev/v1/c/\(projectId)"
+  /**
```

Comment:
was it intentional to reformat the file?

Reply:
Swiftlint fixes, and moving all methods inside the module class.